chart/values.yaml

```diff
@@ -352,6 +352,10 @@ csi:
     priorityClassName: ""
     # -- Prevent modifying the volume mode when creating a PVC from an existing VolumeSnapshot
     preventVolumeModeConversion: true
+    # Enable auto garbage collection of volume resources which were associated with `Retain` PV's.
+    # Before enabling this, make sure you're well aware of the dangerous downsides.
+    # For more information see: <https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain>.
+    enableDangerousRetainGC: false
```

Comment:
Is naming ok @Abhinandan-Purkait  @abhilashshetty04  @niladrih 
Should it be:
`enableDangerousVolGC` ?

Reply:
I would say the description is good enough, we can emit the word dangerous, or you can probably make the doc comment show up on the values table. How about `enableRetainVolGC`?

Reply:
Discussed offline, will merge as is.